samples/mjpeg_usb_cam/docker-compose.l4t.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.3"
+services:
+
+  usb-cam:
+    image: ghcr.io/insight-platform/savant-adapters-gstreamer-l4t:latest
+    restart: unless-stopped
+    volumes:
+      - zmq_sockets:/tmp/zmq-sockets
+    environment:
+      - URI=/dev/video0
+      - FFMPEG_PARAMS=input_format=mjpeg,video_size=1920x1080
```

Comment:
It's probably better to use 720p by default.

Reply:
Does not work on my cam.